release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -1208,77 +1214,6 @@ This script removes unauthenticated subjects from the following cluster role bin
 +
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1821771[*BZ#1821771*])
```

Comment:
Question @mjpytlak : Should that bugzilla citation also be removed? Granted, this appears not to be part of your removed text but thought I should ask the question, it's just kind-of hanging there in the preview by itself.

Reply:
> Should that bugzilla citation also be removed? Granted, this appears not to be part of your removed text but thought I should ask the question, it's just kind-of hanging there in the preview by itself.

@GroceryBoyJr Are you referrring to the BZ link at line 1218. If so, that link is for the existing `oc anotate`  known issue. What you do ;mean by hanging in the preview by itself?

Reply:
Yes, that link. It is offset to the left, so visually, it appears not to refer to the information above it. If you meant to leave it alone (and it appears you did), then I'm happy leaving it alone also. (I'm out-of-scope by pointing out something you did not edit.)